Add Nick Image

KSNINJA opened the pull request:
Added the Nick images to the Mentors and Students section
Included Screenshots of the changes in the mentors.yml and students.yml .
Added nick images to img/mentors and img/students

Link to the repo - https://github.com/KSNINJA/gci18.fossasia.org
Link to the hosted website. - https://ksninja.github.io/gci18.fossasia.org/


![](https://user-images.githubusercontent.com/26023139/47544776-5c843700-d906-11e8-8cd1-6c5925071aa5.png)

![](https://user-images.githubusercontent.com/26023139/47544797-7291f780-d906-11e8-8388-ba9a51d1047a.png)

abishekvashok commented on CNAME:
```diff
@@ -1 +0,0 @@
-gci18.fossasia.org
```
Please don't delete it please.

abishekvashok commented on _data/students.yml:
```diff
@@ -11,4 +11,10 @@
\ No newline at end of file
+  image: nick.png
+
+
```
Please remove these empty spaces.

KSNINJA commented:
I have committed the changes
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/26023139/47546849-aa9e3800-d910-11e8-9fa1-a0ded5fa2083.png">
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/26023139/47546872-c99cca00-d910-11e8-971d-a550273757ab.png">

KSNINJA commented:
But I have changed only 2 files - students.yml and mentors.yml
And I have also deleted the .DS_store file

fragm3 commented:
@KSNINJA this PR is for adding Nick Image, you have to add mentor in a separate issue(Task). Also, resolve the conflicts, squash your commits.

KSNINJA commented:
Please guide me . I am getting confused . What should I do next ?
I have updated the mentors.yml and students.yml and added the nick images . I have also deleted the .DS_Store files

simsausaurabh commented:
Try to squash commits and show clearly whatever changes you are making.

tabesin commented:
Please add .DS_Store to your .gitignore

KSNINJA commented:
This is the students.yml . 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/26023139/47616051-88a5e080-dadd-11e8-91d6-6eaf70382dd7.png">
This is the students.yml file . Even when I remove the red highlighted statement and other unwanted symbols. The same error is thrown and students.yml file automatically changes to this state again .
